jsontest/src/main/java/MainTemp.java

```diff
@@ -0,0 +1,28 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+public class MainTemp {
+    public static void main(String[] args) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+                System.out.println("Enter the city: ");
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
Way easier is to do with Scanner

Reply:
yes, but I read that in principle it is better with BufferedReader